engine/render/wscript

```diff
@@ -30,15 +31,16 @@ def configure(conf):
 
     conf.recurse('src')
 
+    target_os = conf.env['TARGET_OS']
     platform = conf.env['PLATFORM']
 
-    if platform in "linux":
+    # TODO: check if here was mistake - if platform in "linux":
+    if target_os == TargetOS.LINUX:
```

Comment:
Q: There was no platform called `linux`. I guess it was bug here and fix it with `target_os ==`. Is it correct?

Reply:
Looks like an old bug yes. "linux" (32 bit) was removed some years ago.
I guess this code path wasn't used for that period of time, which suggests it isn't needed now either?